src/supertux/main.cpp

```diff
@@ -366,8 +366,10 @@ Main::launch_game()
 
   timelog("audio");
   SoundManager sound_manager;
-  sound_manager.enable_sound(g_config->sound_enabled);
-  sound_manager.enable_music(g_config->music_enabled);
+  sound_manager.enable_sound(g_config->sound_volume > 0 ? true : false);
+  sound_manager.enable_music(g_config->music_volume > 0 ? true : false);
```

Comment:
An alternative might be to have two buttons left and right (currently: < 120 > , then  < < 120 > > ), and the first and last could be used for instant toggling to highest/lowest.